### DISCUSSION TITLE
RenovateBot documentation about fine-grained personal access tokens is out of date

### DISCUSSION THREAD

**Discussion by jsoref** [renovatebot/renovate#27336] [Request Help] [answered]
### How are you running Renovate?

None

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

The documentation is stale.

See:
* https://github.com/renovatebot/renovate/issues/27335

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by viceice** [comment, marked as answer]
PR welcome 🤗

**Reply by jsoref** [reply]
Will you open a ticket so I can hang my PR from it?

Your repo is kinda hostile to contributors.

**Reply by viceice** [reply]
we don't need an issue, simply reference this discussion.
you can also reference the original issue if you like.

**Reply by jsoref** [reply]
PR submitted: https://github.com/renovatebot/renovate/pull/27346
